docs/src/content/docs/it/getting-started.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Iniziamo
+description: Impara come iniziare a costruire il tuo prossimo sito per documentazione con Starlight per Astro.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+:::caution[Lavoro in corso]
+Starlight è in fase di sviluppo iniziale.
+Se trovi qualcosa che non funziona, [apri una nuova issue su GitHub](https://github.com/withastro/starlight/issues/new/choose) o contattaci su [Discord](https://astro.build/chat).
+:::
+
+## Creare un nuovo progetto
+
+Starlight è un tema per documentazione completo di funzionalità costruito su [Astro](https://astro.build).
+
+Puoi creare un nuovo progetto Astro + Starlight usando il comando:
+<Tabs>
+<TabItem label="npm">
+
+```sh
+# crea un nuovo progetto con npm
+npm create astro@latest -- --template starlight
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+# crea un nuovo progetto con pnpm
+pnpm create astro --template starlight
+```
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+# crea un nuovo progetto con yarn
+yarn create astro --template starlight
+```
+
+</TabItem>
+</Tabs>
+
+Questo creerà una nuova [cartella del progetto](/it/guides/projet-structure/) con tutti i file e configurazioni necessarie per il tuo sito.
+
+:::tip[Guardalo in azione]
+Prova Starlight nel browser:
+[apri il template su StackBlitz](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics).
+:::
+
+## Creare contenuti con Starlight
+
+Starlight è pronto per aggiungere nuovi contenuti, compresi i tuoi file!
+
+### Formati dei file
+
+Starlight supporta la creazione di contenuti in Markdown e MDX. (Puoi aggiungere il supporto per Markdoc installando l'integrazione sperimentale [Astro Markdoc](https://docs.astro.build/fr/guides/integrations-guide/markdoc/)).
```

Comment:
Credo che sarebbe meglio usare i link alla documentazione inglese invece di francese (finché non abbiamo pure la documentazione di Astro in italiano 😅):

```suggestion
Starlight supporta la creazione di contenuti in Markdown e MDX. (Puoi aggiungere il supporto per Markdoc installando l'integrazione sperimentale [Astro Markdoc](https://docs.astro.build/en/guides/integrations-guide/markdoc/)).
```